frontend/src/pages/secret-manager/SecretDashboardPage/components/ActionBar/FolderForm.tsx

```diff
@@ -61,6 +89,31 @@ export const FolderForm = ({
           </FormControl>
         )}
       />
+      <Controller
+          control={control}
+          name="description"
+          defaultValue=""
+          render={({ field, fieldState: { error } }) => (
+            <FormControl
+              label="Folder Description"
+              isError={Boolean(error)}
+              tooltipText={showDescriptionOverwriteWarning ? descriptionOverwriteWarningMessage : undefined}
+              isOptional
+              errorText={error?.message}
+              className="flex-1"
+            >
+              <TextArea
+                placeholder="Folder description"
+                {...field}
+                rows={3}
+                ref={descriptionRef}
+                onInput={handleInput}
+                className="thin-scrollbar w-full !resize-none bg-mineshaft-900"
+                maxLength={255}
+              />
+            </FormControl>
+          )}
+        />
```

Comment:
if a folder already has a description and I intend to remove the description by setting it to "", it would retain the description instead of remove it 
<img width="1510" alt="image" src="https://github.com/user-attachments/assets/14e90808-2bd4-466e-8fef-4c7a174ab299" />

<img width="720" alt="image" src="https://github.com/user-attachments/assets/9dfc6b7e-5f1b-40a2-ade3-0a369ddbb007" />